src/containers/userChannels/ChannelsContainer.jsx

```diff
@@ -137,7 +143,21 @@ class ChannelsContainer extends Component {
     this.props.pollTokens();
   };
 
+  hasChannelOpened() {
```

Comment:
i think this should be called `isChannelOpen` instead, but it's not a blocker

Reply:
actually we are talking about a list of channels, and the name of the method refers to that list of channels, maybe we can change that to `hasOpenChannel` or something like that.

Reply:
oh, my bad. thanks for the clarification

Reply:
(either of the names you suggest is fine by me, but `hasOpenChannel` is definitely clearer)